### DISCUSSION TITLE
Puan

### DISCUSSION THREAD

**Discussion by Yunus044** [IBAX-io/go-ibax#1482] [Soru Cevap - Türkçe]
![Screenshot_20220130_112349](https://user-images.githubusercontent.com/98660264/151692447-c98f5172-2ed9-40d3-a71a-61f6b75c6602.jpg)

Bu niye sıfır bı yardimvi olurmusunuz

**Comment by Skips1** [comment]
Birisi refarans yeri getirdiğin insanlar için kiseyi getirememissin :(

Diğerkisi daha belli değil herkesde 0 kanka her halde son gün puan dağıtımı yapıcaklar en azından ben öyle düşünüyorum.

**Comment by Yusufknk** [comment]
Github puanları, githubta herhangi bir öneri ve fikrinizi paylaştığıniz da ve ibax yöneticileri fikriniz beğenip puan verdiği zaman github puan kısmından görünüyor. Referans parası ise davet linkinizle getirdiğiniz her kişi için ibxc coini alırsınız .
